File: src/main/java/com/willmadison/legacycodekatas/fulfillment/orders/DefaultOrderService.java
package com.willmadison.legacycodekatas.fulfillment.orders;

import java.util.HashSet;
import java.util.Set;

public class DefaultOrderService implements OrderService {

    @Override
    public Set<Order> find(SearchParameters searchParameters) {
        return new HashSet<>();
    }

    @Override
    public void save(Order order) {

    }

    @Override
    public void save(OrderItem orderItem) {

    }
}

File: src/main/kotlin/com/willmadison/legacycodekatas/fulfillment/orders/DefaultOrderService.kt
package com.willmadison.legacycodekatas.fulfillment.orders

class DefaultOrderService : OrderService {

    override fun find(searchParameters: SearchParameters): Set<Order> = emptySet()

    override fun save(order: Order) {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }
}